src/app/components/CareHygieneServices/CareHygieneServices.tsx

```diff
@@ -52,12 +51,24 @@ export const CareHygieneServices: React.FC<CareHygieneServicesProps> = ({
                 <img
                   src={service.src1}
                   alt={service.alt}
-                  className={`absolute inset-0 h-full w-full object-cover transition-opacity duration-1000 ease-in-out ${activePicture ? "opacity-100" : "opacity-0"}`}
+                  className={cn(
+                    "absolute inset-0 h-full w-full object-cover transition-opacity duration-1000 ease-in-out",
+                    {
+                      "opacity-100": activePicture,
+                      "opacity-0": !activePicture,
+                    },
+                  )}
                 />
                 <img
                   src={service.src2}
                   alt={service.alt}
-                  className={`absolute inset-0 h-full w-full object-cover transition-opacity duration-1000 ease-in-out ${activePicture ? "opacity-0" : "opacity-100"}`}
+                  className={cn(
+                    "absolute inset-0 h-full w-full object-cover transition-opacity duration-1000 ease-in-out",
+                    {
+                      "opacity-0": activePicture,
+                      "opacity-100": !activePicture,
+                    },
+                  )}
```

Comment:
I've identified some opportunities for improvement that can enhance the component's performance and maintainability. Please consider the following suggestions:

### **1. Extract Image Switching Logic into a Separate Component:**

To optimize performance, we can create a **separate client component** for the image switching logic. This will prevent unnecessary re-renders of the entire `CareHygieneServices` component every 3 seconds.  
By doing this, we can convert the main `CareHygieneServices` component into a **server component**, removing client-side hooks like `useState` and `useEffect` from it.


### **2. Implement Next.js `Image` Component:**

I suggest replacing the standard `<img>` tags with Next.js `Image` components. This change will bring several benefits:
- **Automatic image optimization**
- **Responsive image sizes**
- **Improved lazy loading**

These improvements should enhance page load speed, especially on slower networks, and potentially boost our SEO by improving **Core Web Vitals**.


### **3. Simplify Image Display Logic:**

Instead of rendering two separate `<img>` tags and toggling their visibility, we propose using a single `Image` component and dynamically updating its `src` attribute.  
This approach will reduce the number of rendered elements and simplify our codebase.


### **4. Refine State Management for Image Switching:**

Rather than using a boolean state for image toggling, a **numeric state** can be used to store the active image index.  
This change will make our code more flexible and easier to expand if we need to add more images in the future.


### **5. Optimize Initial Load with `priority` Prop:**

For the first image in the list, please add the `priority` prop to the `Image` component.  
This will ensure that the first image loads immediately without being deferred, which is particularly important for **above-the-fold** content.